action.yml

```diff
@@ -26,7 +32,7 @@ inputs:
       A pre-checked out collection directory that's already on disk,
       substitutes getting the source from the remote Git repository if
       set. This action will not attempt to mutate its contents
-    default:
+    default: auto
```

Comment:
@webknjaz hmm, it looks like something went wrong during rebasing. `auto` should be the default of `controller-python-version`, and not of `collection-src-directory`.